### DISCUSSION TITLE
just hide navbar in appshell

### DISCUSSION THREAD

**Discussion by CFarhad** [mantinedev/mantine#3395] [Q&A] [answered]
i want to hide the navbar only but when i use `hidden` property in the Appshell component it hides all things except main section.
its a general feature in all dashboard panels. how can i develop this without a drawer

**Comment by rtivital** [comment]
https://codesandbox.io/s/musing-mendeleev-hk9qbl?file=/src/App.tsx

**Comment by CFarhad** [comment, marked as answer]
tnk u ❣️. but, how can i use a transition on it ???

i tested this but it is not still work 

```jsx
<Transition  mounted={opened} transition="scale-x" duration={400} timingFunction="ease" >
    <Navbar width={{ base: 200 }}>Navbar</Navbar>
</Transition>
```

also in this code transition doesn't work
```jsx
<Transition  mounted={opened} transition="scale-x" duration={400} timingFunction="ease" >
          {(styles) => (
            <Navbar  width={{ base: 200 }}>Navbar</Navbar>
          )}
</Transition>
```

**Reply by Sajarin-M** [reply]
Try this
```js
<Transition  mounted={opened} transition="scale-x" duration={400} timingFunction="ease" >
    {(styles) => (
        <Navbar style={styles} maw={{ base: 200 }}>Navbar</Navbar>
    )}
</Transition>
```

**Reply by CFarhad** [reply]
it works. but the main content doesn't work smoothly. do you have any idea for it with Transition or framer motion?

**Comment by Sajarin-M** [comment]
What seems to be the issue with the main content?
Additionally, if you are looking to achieve a similar effect using Framer Motion, I recommend checking out [this sandbox](https://codesandbox.io/s/framer-motion-hju040?file=/src/App.tsx) for reference.
